### ISSUE TITLE
glady

### ISSUE THREAD

**Issue by glady-anne** [InvexGames/MaterialUI#56]
hi! it seems im facing the same issue with Assets/MaterialUI/Scripts/ShadowGen.cs(292,40): error CS0117: `System.IO.File' does not contain a definition for `WriteAllBytes' 

how can i fix it ? any solution please ?

**Comment by nah0y** [comment]
Hello,
What platform are you targeting?

**Comment by glady-anne** [comment]
android . can you help me how to fix it ?

**Comment by nah0y** [comment]
What version of Unity are you using?

**Comment by NurulUs** [comment]
using unity version 5.2.0
Pada tanggal 6 Jun 2016 22.23, "Yohan Hadjedj" <notifications@github.com>
menulis:

**Comment by nah0y** [comment]
Hum... are you sure you're on Android?
Seems like you're targeting WebPlayer that is not supported.

Go to File -> BuildSettings and switch to Android platform.

**Comment by NurulUs** [comment]
Yes, using for my android.
Pada tanggal 7 Jun 2016 22.55, "Yohan Hadjedj" <notifications@github.com>
menulis:

**Comment by glady-anne** [comment]
@nah0y  the unity version is 5.3 . should i download only the beta ?

**Comment by glady-anne** [comment]
yep i already download the android platform thanks @nah0y  . now my new problem is when i click ADD OPEN SOURCES i go to the  folder where i put what im download. then the android platform is .EXE but the only choices there to import is .UNITY and ALL FOLDERS? question is how can i import the android platform ?

**Comment by nah0y** [comment]
I don't understand a single thing of what you said, sorry lol
